src/pybind/mgr/dashboard/frontend/src/styles/bootstrap-extends.scss

```diff
@@ -21,7 +21,7 @@ cd-info-card {
 }
 
 .btn-toolbar cd-table-actions.btn-group {
-  @extend .me-2;
+  @extend .me-0;
```

Comment:
This issue is specifc to a component only, perhaps we can try to check how other components are aligned and not this. You can check other tables is pages like -OSD, Pools, Hosts, etc.

The issue is not there but here, and if we change global css this might effect other components in unexpected ways (scary!)

Can you check ?